photon-lib/py/setup.py

```diff
@@ -0,0 +1,17 @@
+from setuptools import setup, find_packages
+import subprocess
+
+versionStr = subprocess.check_output(['git', 'describe', '--tags', "--match=v*", "--always"]).decode('utf-8').strip()
+print(f"Building version {versionStr}")
+
+setup(
+    name='photonlibpy',
+    packages=find_packages(),
+    version="0.0.2",
+    install_requires=[
+        "wpilib<2025,>=2024.0.0b2", 
```

Comment:
I think you're missing a few direct dependencies.

Reply:
hmmm. So I'll admit, I just copied this dependency line and it seems to ... "work" ?

The direct dependencies I think I have are ntcore, wpimath, and wpilib itself.... I assume by standard those should all be pulled in directly (Or, furthermore.... don't assume wpilib is going to pull them in?)